ui/webui/resources/polymer_overriding.js

```diff
@@ -18,6 +18,8 @@ if (debug) {
 }
 
 const allBehaviorsMap = {}
+const allPropertiesMap = {}
+const missingComponentsMap = {}
```

Comment:
I think this is hard to understand and follow what it's used for.

How about calling it something explicit like `componentPropertyModifications` or `propertyModificationComponentBuffer` and a comment stating that components are held there in case `RegisterPolymerComponentProperties` is called after the component is registered?